pulumitest/preview.go

```diff
@@ -0,0 +1,21 @@
+package pulumitest
+
+import (
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/pulumi/pulumi/sdk/v3/go/auto/optpreview"
+)
+
+// Preview previews an update to the current stack.
+func (a *PulumiTest) Preview(opts ...optpreview.Option) auto.PreviewResult {
```

Comment:
If we're doing wrappers, do wrappers all the way here.. Wrap auto.PreviewResult so we can add helper methods at the level of this library.

Reply:
Just an idea.

Reply:
Maybe, though this increases the implementation size and adds more elements to keep in-sync with the underlying implementation. I'd delay putting this in for the return types for now and re-evaluate if we hit any messy cases as-is.